web/tests/firestore/firestore.test.js

```diff
@@ -0,0 +1,22 @@
+import { auth } from '../../firebase';
+
+let user;
+
+beforeAll(() =>
+  auth.createUserWithEmailAndPassword('user@example.com', '12345678').then((userCredential) => {
+    console.log('user successfully created!');
+    ({ user } = userCredential);
+  }).catch((error) => {
+    console.err('failed to create user: ', error);
+  }));
+
+test('basic test', () => {
+  expect(user.email).toBe('user@example.com');
+});
+
+afterAll(() =>
+  user.delete().then(() => {
+    console.log('user successfully deleted!');
```

Comment:
it's probably best not to pollute test output with successes - the `console.err` on failures should be sufficient (if it didnt fail, then it succeeded)